[2023-03-29 15:55:44,894] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:55:44.094028+00:00 [queued]>
[2023-03-29 15:55:44,909] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:55:44.094028+00:00 [queued]>
[2023-03-29 15:55:44,909] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:55:44,909] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 15:55:44,910] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:55:44,932] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): start_task> on 2023-03-29 15:55:44.094028+00:00
[2023-03-29 15:55:44,939] {standard_task_runner.py:52} INFO - Started process 171 to run task
[2023-03-29 15:55:44,942] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'start_task', 'manual__2023-03-29T15:55:44.094028+00:00', '--job-id', '608', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmpjvyfl64o', '--error-file', '/tmp/tmpyagqduq4']
[2023-03-29 15:55:44,942] {standard_task_runner.py:80} INFO - Job 608: Subtask start_task
[2023-03-29 15:55:45,011] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:55:44.094028+00:00 [running]> on host 4513f77b4fd8
[2023-03-29 15:55:45,113] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=start_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T15:55:44.094028+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T15:55:44.094028+00:00
[2023-03-29 15:55:45,114] {logging_mixin.py:115} INFO - DAG has been started...
[2023-03-29 15:55:45,115] {python.py:173} INFO - Done. Returned value was: DAG has been started...
[2023-03-29 15:55:45,149] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=start_task, execution_date=20230329T155544, start_date=20230329T155544, end_date=20230329T155545
[2023-03-29 15:55:45,196] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 15:55:45,255] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
